test/specs/gfm/gfm-spec.js

```diff
@@ -15,7 +15,8 @@ Messenger.prototype.test = function(spec, section, ignore) {
     var shouldFail = ~ignore.indexOf(spec.example);
     it('should ' + (shouldFail ? 'fail' : 'pass') + ' example ' + spec.example, function() {
       var expected = spec.html;
-      var actual = marked(spec.markdown, { headerIds: false, xhtml: true });
+      var usexhtml = typeof spec.xhtml === 'boolean' ? spec.xhtml : true;
+      var actual = marked(spec.markdown, { headerIds: false, xhtml: usexhtml });
```

Comment:
It looks like just setting the `xhtml` option to false for all gfm the tests works

Reply:
I think CommonMark uses XHTML maybe the GFM extensions don't need XHTML. (I do think there's something to be said for the attempted solution - putting the options in the JSON - but not sure we've hit a point of actually needing it.)